### DISCUSSION TITLE
emailVisibility...뭔지 모르겠어요..🫠

### DISCUSSION THREAD

**Discussion by kimInDa** [FRONTENDSCHOOL6/react-project-11#63] [Q&A] [answered]
pocketbase의 email 필드에 설정하는 옵션인데,
![2023-09-15_01-25-42 png](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/105577805/0a804409-5c06-42d9-876f-6df0b664eea3)
![2023-09-15_01-25-56 png](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/105577805/9de01869-371a-411a-aafd-0ace5c957f2c)

공식문서 설명을 봐도 뭐하는건지 모르겠어요..🥲
![2023-09-15_01-52-31](https://github.com/FRONTENDSCHOOL6/react-project-11/assets/105577805/0a8bd21b-1135-4218-a68c-6aec83aef45c)

**Comment by ryudg** [comment, marked as answer]
https://github.com/pocketbase/pocketbase/discussions/1442

다인님께서 주신 질문과 같은 질문을 남겨 주신 분이 있어 링크 남겨봅니다.

시스템에서 사용자의 개인 정보 공개 범위를 관리하는데 중요한 역할을 하는데, 일부 사용자는 자신의 이메일 주소가 다른 사용자에게 보여지길 원하지 않을 수 있기 때문에 존재하는 것으로 판단됩니다.
이 기능을 pocketbase에서 기본 제공해주어 개발자가 직접 기능 구현을 하지 않도록 도와주는 것 같습니다.

**Reply by summer787** [reply]
아하 덕분에 알게 되었습니다 감사합니다!
